tests/test_tally.py

```diff
@@ -278,18 +277,6 @@ def test_tally28():
 
     assert_equal(tally.entity_size,new_tally.entity_size)
 
-def test_tally28():
```

Comment:
Why did we get rid of this test?

Reply:
my mistake, this was not intended

Reply:
Actually, it was an empty test that was just hanging there. (the real `test_tally28()` is defined L268)

I just cleaned this up....

Reply:
It doesn't seem completely empty... but I'm not sure if/how it works...

Reply:
yes you are write... sorry my bad, 2 tests had same labels, and I overlook at them to quickly... I put it back....

Reply:
maybe we should rename the second one tho....